### DISCUSSION TITLE
why doesnt ./eww open <my window name which is example> doesnt work?

### DISCUSSION THREAD

**Discussion by itzRealDuck** [elkowar/eww#727] [Q&A]
e

**Comment by elkowar** [comment]
I don't know.

see also: [how do I ask a good question?](https://stackoverflow.com/help/how-to-ask)

**Comment by webbb82** [comment]
man I dont know what im doing wrong but that dang example code. The first part works no problem but once I add that second part . the whole thing breaks. No matter where I add it. It says that its suppose to go in place of the text no matter if i put it in the quotes or over them or whatever .

**Reply by mantacid** [reply]
Can you show the minimal config required to replicate the behavior you're experiencing? That might help us to help you.

**Comment by mantacid** [comment]
From what you wrote in the question, you’re using `./eww` as the location of the executable. If your config is split amongst multiple directories, this might not work. Use `${EWW_EXECUTABLE}` instead to be able to use the command in yuck files that are in different directories than the main config.

**Reply by mantacid** [reply]
Alternatively, symlink the built executable into the system folder that contains executables ( `/usr/bin` perhaps? I don't remember what I did). That allows you to run eww no matter where you call it just by using `eww [STUFF]`

**Reply by Souliboi** [reply]
~~From what they are saying it can be suggested to be the similar issue I am facing which quite literally is the documentation, or my competence [failing](https://github.com/elkowar/eww/issues/909)~~

EDIT: Solved, i am blind

**Reply by mantacid** [reply]
Well what was your solution?

**Reply by mantacid** [reply]
> Alternatively, symlink the built executable into the system folder that contains executables ( `/usr/bin` perhaps? I don't remember what I did). That allows you to run eww no matter where you call it just by using `eww [STUFF]`

Actually, it should be symlinked to `/usr/local/bin/eww`. **ESPECIALLY** on arch-based distros.
